plugin.rb

```diff
@@ -127,4 +127,45 @@ def retorts
       return ActiveModel::ArraySerializer.new(Retort::Retort.for_post(post: object), each_serializer: ::Retort::RetortSerializer).as_json
     end
   end
+
+  require_dependency 'rate_limiter'
+  require_dependency 'post_detail'
+  class ::PostDetail
+    include RateLimiter::OnCreateRecord
+    rate_limit :retort_rate_limiter
+    after_update :retort_after_update
+
+    def is_retort?
+      extra == RETORT_PLUGIN_NAME
+    end
+
+    def retort_after_update
```

Comment:
Hmmmm copy paste from OnCreateRecord doesn't feel great for on_update, I wonder if you'd be willing to try something like
```
after_update { run_callbacks :create if is_retort? }
```
That should run both `before_create` and `after_create`, but only for retorts, so if that line works it feels much cleaner to me.

Reply:
Yup that works 👍 

https://github.com/gdpelican/retort/pull/38/commits/564a20877379697b0137b9dd04b71b48730c020d